main.go

```diff
@@ -77,13 +77,17 @@ func main() {
 	var probeAddr string
 	var syncPeriod time.Duration
 	var logLevel string
+	var rdsControllerRetries int
+	var rdsControllerInterval time.Duration
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	flag.DurationVar(&syncPeriod, "sync-period-min", 180*time.Minute, "The minimum interval at which watched resources are reconciled (e.g. 30 minutes)")
+	flag.DurationVar(&syncPeriod, "sync-period-min", 180*time.Minute, "The minimum interval at which watched resources are reconciled (e.g. 30 minutes).")
 	flag.StringVar(&logLevel, "log-level", "info", "Log level.")
+	flag.IntVar(&rdsControllerRetries, "wait-for-rds-controller-retries", 15, "The maximum times to check if the RDS controller is ready to run before setting up the Inventory controller.")
```

Comment:
do we required to add [here](https://github.com/RHEcosystemAppEng/rds-dbaas-operator/blob/main/config/manager/manager.yaml#L65) in env variable WDYT ?

Reply:
I don't think so, the default values should work in most cases.